src/functional_test_runner/lib/config/schema.js

```diff
@@ -88,6 +88,9 @@ export const schema = Joi.object().keys({
     elasticsearch: urlPartsSchema(),
   }).default(),
 
+  kibanaServerArgs: Joi.array(),
+
+  // env allows generic data, e.g. for fields unrelated to URLs
   env: Joi.object().default(),
```

Comment:
__note:__ I know without comment it was even worse, but I still don't get what `fields unrelated to URLs` means?

Reply:
I have no idea 🙈  I don't know what this is for tbh, or how to explain it.